### ISSUE TITLE
Returns wrong word count

### ISSUE THREAD

**Issue by fahadonline** [haakym/word-count#1]
It seems to return wrong word count for following doc file I tested.
Please check if the issue is reproduced on your side.

[magento_2.doc.zip](https://github.com/haakym/word-count/files/1131169/magento_2.doc.zip)

**Comment by haakym** [comment]
@fahadonline 

Yeah I have had some issues with it performing the correct count. This package needs some fine tuning for sure. You're welcome to submit a PR and tests if you can find a solution!

You could try this package out: https://github.com/joeblurton/doccounter

Hope this helps!

**Comment by fahadonline** [comment]
@haakym 

Thanks for reply.
